android/src/main/java/com/learnium/RNDeviceInfo/RNInstallReferrerClient.java

```diff
@@ -75,7 +75,7 @@ private String getInstallReferrer() {
       @Override public void onInstallReferrerServiceDisconnected() {
         // Documentation indicates the InstallReferrer connection will be maintained
         // So there is really nothing to do here
-        if (BuildConfig.DEBUG) Log.d("RNInstallReferrerClient", "InstallReferrerService disconnected");
+        //if (BuildConfig.DEBUG) Log.d("RNInstallReferrerClient", "InstallReferrerService disconnected");
```

Comment:
```suggestion
        Log.d("RNInstallReferrerClient", "InstallReferrerService disconnected");
```

Reply:
Yes, this works :-)

When you plan to merge the change? I will try to pull and test the build 

Thanks a lot for your help